rclcpp/include/rclcpp/macros.hpp

```diff
@@ -31,11 +31,11 @@
 #define RCLCPP_MAKE_SHARED_DEFINITIONS(Class) \
   typedef std::shared_ptr<Class> SharedPtr; \
   \
-  template<typename...Args> \
+  template<typename ... Args> \
   static std::shared_ptr<Class> \
-  make_shared(Args &&...args) \
+  make_shared(Args && ... args) \
   { \
-    return std::make_shared<Class>(std::forward<Args>(args)...); \
+    return std::make_shared<Class>(std::forward<Args>(args) ...); \
```

Comment:
I assume you have or will run the farm over these changes, so this will come up then if it is a problem.